src/main/core/support/options.c

```diff
@@ -385,11 +378,6 @@ gboolean options_shouldExitAfterShmCleanup(Options* options) {
     return options->shouldExitAfterShmCleanup;
 }
 
-const gchar* options_getPreloadString(Options* options) {
```

Comment:
Huh, this wasn't called anywhere?

Reply:
Nope, it looks like all uses were removed in the initial e41c4a997.